layouts/_default/list.html

```diff
@@ -31,7 +31,11 @@
 
 {{ $paginator := .Paginate (where .Data.Pages "Type" "post") }}
 {{ range $paginator.Pages }}
+{{ if (ne ($.Param "truncate") false) }}
```

Comment:
So I think changed this such that should make this settable either in config.tom or in post front-matter. After reading the "Param method" section of https://gohugo.io/templates/variables/, it doesn't seem like there should be anything to go wrong. I'm unable to verify the front-matter override in my manual tests, though.

* Setting truncate via the [params] section of config.tom works as expected (truncate if unset, truncate if true, full-content if false).
* Setting trucate via post front-matter seems to have no effect no matter what I do.

This is my first foray in to Go, Go templating, and Hugo theme development, and I'm at a bit of a loss how to debug this seemingly internal Hugo behavior. I'm posting the postentially broken PR in the hope that many eyeballs make bugs shallow.